indy_common/test/auth/test_auth_error.py

```diff
@@ -0,0 +1,23 @@
+from plenum.common.constants import NODE, TRUSTEE, BLS_KEY, STEWARD
+
+from indy_common.auth import Authoriser
+
+
+def test_node_not_allowed_role_error():
```

Comment:
Is there test for successful auth?

Reply:
Yes, recently added tests in `indy_common/test/auth` package (test_auth_node, test_auth_nym, etc.).